crates/loans/src/lib.rs

```diff
@@ -297,63 +297,125 @@ pub mod pallet {
     #[pallet::generate_deposit(pub (crate) fn deposit_event)]
     pub enum Event<T: Config> {
         /// Enable collateral for certain asset
-        /// [sender, asset_id]
-        DepositCollateral(T::AccountId, AssetIdOf<T>, BalanceOf<T>),
+        DepositCollateral {
+            account_id: T::AccountId,
+            currency: AssetIdOf<T>,
```

Comment:
Minor nit: would rename these to `asset_id` or `currency_id` probably

Reply:
I'll use `currency_id` for consistency with the other pallets